client/src/panels/panel_declarations/finances/detailed_program_spending_split/detailed_program_spending_split.js

```diff
@@ -251,6 +251,7 @@ class DetailedProgramSplit extends React.Component {
       .sortBy("total")
       .value();
 
+    // Change height of the graph region according to the number of programs
```

Comment:
Hmm, sorry to nitpick, but this is more of the "what", I was thinking more of the "why". Maybe add a second sentence saying something like "This is required for y-axis labels to have sufficient room when many programs are present".